src/Components/TimeLines.js

```diff
@@ -0,0 +1,23 @@
+import TasksList from './TasksList'
+const TimeLines = ({listTodoToday,listTodoTomorrow,listTodoUpcoming}) => {
+    return ( 
+        <div className="timeLines">
+            <div className="Today">
+                <h2>Today</h2>
+                <TasksList id="1" listTodoToday={listTodoToday} listTodoTomorrow={listTodoTomorrow} listTodoUpcoming={listTodoUpcoming} />
```

Comment:
Why do we need separate props like `listTodoToday`, `listTodoTomorrow`, and `listTodoUpcoming`? We can simplify it by having a single prop named `taskListData`, for example, and pass the relevant task list each time.